common/src/main/java/com/wynntils/screens/maps/widgets/SeaskipperTravelButton.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.screens.maps.widgets;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.components.Models;
+import com.wynntils.models.map.pois.SeaskipperDestinationPoi;
+import com.wynntils.screens.base.widgets.WynntilsButton;
+import com.wynntils.utils.mc.TooltipUtils;
+import com.wynntils.utils.render.FontRenderer;
+import com.wynntils.utils.render.RenderUtils;
+import com.wynntils.utils.render.Texture;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+
+public class SeaskipperTravelButton extends WynntilsButton {
+    private List<Component> tooltip;
+    private SeaskipperDestinationPoi selectedPoi = null;
```

Comment:
This doesn't belong here. Store the selected poi in the screen class, and pass the screen to this widget. Then you can call a method in the screen, like `travelToDestination`.